src/codecov-clojure/orb.yml

```diff
@@ -46,6 +46,7 @@ executors:
 
 jobs:
   code-coverage:
+    description: Run lein cloverage, save cache and use codecov to upload coverage results
```

Comment:
can we change to "using cache"  or omit. It is confusing to read as saving cache which is a 2 step process (restore, then save)

> Run lein cloverage and upload results to Codecov